### DISCUSSION TITLE
Github action fails: Error: Process completed with exit code 1

### DISCUSSION THREAD

**Discussion by aditodkar** [community/community#25299] [Actions]
<p>I am running workflow but getting this message: Error: Process completed with exit code 1. in npm run build step</p>
<pre><code class="lang-auto">name: Node.js CI

on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]

jobs:
  build:

    runs-on: ubuntu-20.04

    steps:
    - uses: actions/checkout@v2
    - uses: actions/setup-node@v2
      with:
        node-version: 16.14.0
    - run: npm i
    - run: npm run build

</code></pre>

**Comment by aditodkar** [comment, marked as answer]
<p>Once I added CI=false in env it worked any idea why so?</p>

**Reply by bruce0205** [reply]
in github action `CI` default value is true, which means any warnings are treated as build errors.
so set `CI` as false, those warnings are treated as build success

**Comment by tonywdy** [comment]
Have you solved it? I also met:(

**Reply by tonywdy** [reply]
Error: Process completed with exit code 1.

**Reply by airtower-luna** [reply]
Any non-zero exit code usually means error. You need to figure out what exactly failed and why to then find a solution. Your problem might be completely different. :sweat_smile: 

If you need help figuring that out best create a new thread ("new discussion" button), describe what you're trying to do, and include a link to the job log. :cat:

**Comment by DamengRandom** [comment]
Please also ensure your jest has no warnings during pipeline running:

eg: Jest: "global" coverage threshold for branches (10%) not met: 8.33%

Such as ensure the threshold value is always passed:

you can setup something like below in jest.config.js file:

coverageThreshold: {
    global: {
      branches: 1,
      functions: 1,
      lines: 1,
      statements: 1,
    },
  },
